### DISCUSSION TITLE
小建议 望采纳~~

### DISCUSSION THREAD

**Discussion by fl1906** [Molunerfinn/PicGo#700] [Ideas]
小建议 把下面链接格式选项 设置为 可点击 点击则选择复制哪一个 因为有时候会选择复制多种格式
![image](https://user-images.githubusercontent.com/73702449/126077350-e4f9841b-8ab6-45f6-a4a4-b1b501b841c4.png)

**Comment by Molunerfinn** [comment]
感谢建议，但是有的时候没有上传图片，点击这里复制什么呢？想要复制不同格式的话相册已经实现这个功能了

**Comment by fl1906** [comment]
emmm 就是 相册那个的话 
![image](https://user-images.githubusercontent.com/73702449/126097068-f572c84f-48a8-492c-91b8-74dc43b82a4b.png)
我想的就是 这样子 虽然也能达到 目的 但是 步骤有些繁琐（使用者个人看法。。）没有上传图片就提示 无可复制图片就好了 或者
干脆 无图片就把下面五个链接格式禁用掉.. 个人想法如觉得无必要这样做可忽略 作者大大辛苦了~

**Reply by Molunerfinn** [reply]
这样边界情况太多，而且不符合以往用户习惯。不过我可以考虑开放复制事件，可以让插件来实现点击复制之后的行为，这样会更好一点
